bench/workgen/runner/chunkcache.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
```

Comment:
Should the test be added to Evergreen?

Reply:
Will loop @korteland on this discussion. We have a few tests now for chunkcache.

Reply:
After discussing this offline a bit more - this should be on Evergreen, but on a weekly schedule or something like that. There are race conditions that are more likely to trigger in this than in Python as it can go faster, but it's a long-running test and I want to be conscious of Evergreen resource usage.

Reply:
added here: [b8c7686](https://github.com/wiredtiger/wiredtiger/pull/9507/commits/b8c76869ccd349c63e10f38efb1cc450c2299452)